applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -382,16 +415,35 @@ def _CreateLevelSetConvectionProcess(self):
                 raise Exception("The BFECC level set convection requires the Kratos ConvectionDiffusionApplication compilation.")
         else:
             linear_solver = self._GetLinearSolver()
+            if self._eulerian_error_compensation: # this is used to perform one step BFECC for the eulerian LS convection
+                bfecc_order = 1
+            else:
+                bfecc_order = 0
+
+            max_cfl = 1.0
```

Comment:
I think these should probably go to the parameters. Also, I am not really involved in the design of the fluid solvers, but I think this class is becoming just too big and does too many things. In my opinion, probably it'd be better to move things related to convector to an auxiliary class...

Reply:
I agree @ddiezrod, specially considering that this changes the current behavior. All these must be exposed to the user.

We can even create a new section in the json settings to encapsulate all the level set stuff, namely `level_set_settings` (or something of the sort). In there we can hardcode the most robust defaults, which I think are the ones you put in here.